parquet-avro/src/test/java/org/apache/parquet/avro/TestReadWrite.java

```diff
@@ -426,7 +426,7 @@ public void testAll() throws Exception {
     assertEquals(emptyMap, nextRecord.get("myemptymap"));
     assertEquals(genericFixed, nextRecord.get("myfixed"));
   }
-
+  
```

Comment:
Nit: Please remove the whitespace-only changes. It's harder to pick commits between branches and do maintenance with these changes.